config/routes.rb

```diff
@@ -12,7 +12,7 @@
   # All the Api endpoints must be under /api/v1 and must have an extension .json.
   namespace :api do
     namespace :v1 do
-      resources :sessions, only: %i[index create] do
+      resources :sessions, only: %i[index create destroy] do
         collection do
           delete 'signout', to: 'sessions#destroy'
```

Comment:
I believe that we need to remove the `DELETE /sessions/signout.json` collection route L16-L18 and update the `useDeleteSession` hook to use the `DELETE /sessions.json` API instead.
Because this way the `SessionsController#destroy` is not accessed through the destroy resource API.

Reply:
That was my first though as well - but then I realized `destroy` requires an `id` which doesn't really exist in this context. Looks like I forgot to remove this

Reply:
Gotcha,